src/storage/mvcc/mod.rs

```diff
@@ -1046,7 +1046,14 @@ pub mod tests {
         let for_update_ts = for_update_ts.into();
         let cm = ConcurrencyManager::new(for_update_ts);
         let mut txn = MvccTxn::new(snapshot, start_ts.into(), true, cm);
-        txn.pessimistic_rollback(Key::from_raw(key), for_update_ts)
+        let mut command = crate::storage::txn::commands::PessimisticRollback {
```

Comment:
Construct commands in tests seems strange. I wonder how can we improve this.

Reply:
Can we move these tests to the command files? I think we can test `process_write` rather than `pessimistic_rollback`, etc.

Reply:
> Can we move these tests to the command files? I think we can test `process_write` rather than `pessimistic_rollback`, etc.

The problem of testing `process_write` is that the `must_xxx` jobs depends on each other, and if I move them together, it will make this pr a huge one.
And the same problem prevents me from inline some functions into `process_write`.

Reply:
Hmm, if the PR is large, but its only moving code, then I think its OK. I can't think of a better approach. Even if we land this as is, we'd still have a huge PR when we do that with later refactoring.